js/base-chat.js

```diff
@@ -0,0 +1,770 @@
+			if (mods.class) msg.addClass(mods.class);
+			if (mods.versions) msg.append(playerVersionsTemplate(mods.versions));
+			if (mods.decolon) msg.find(".by").text((i, txt) => txt.replace(/(?:\(To |)(.+?)\)?:/, "$1"));
+			if (mods.action) d20plus.chat.smallActionBtnAdd(msg, mods.action);
+			delete d20plus.chat.modify[id];
+		});
+		/* $(`.message .userscript-modify-message`).each((i, el) => {
+			const msg = { $el: $(el) };
+			msg.body = msg.$el.closest(".message");
+			msg.by = msg.body.find(".by");
+			msg.$el.text().split("|").forEach(p => {
+				const params = p.split(":");
+				if (params.length === 2) msg[params[0]] = params[1];
+			});
+			if (msg.declass) msg.body.removeClass(msg.class);
+			if (msg.class) msg.body.addClass(msg.class);
+			if (msg.undo) msg.body.append(msgActionButtonTemplate({type: "undo"}));
+			if (msg.request) msg.body.append(msgActionButtonTemplate({type: "request", action: msg.request}));
+			if (msg.versions) msg.body.append(playerVersionsTemplate(msg.versions));
+			if (msg.decolon) msg.by.text(msg.by.text().replace(/(?:\(To |)(.+?)\)?:/, "$1"));
+			msg.$el.remove();
+		}); */
+	}
+
+	d20plus.chat.sending = () => {
+		d20plus.chat.resetSendMyself();
+		const tc = d20.textchat.$textarea;
+
+		if (d20plus.cfg.getOrDefault("chat", "emoji")) {
+			tc.val(tc.val().replace(/(:\w*?:)/g, (m0, m1) => {
+				const clean = m1.replace(/:/g, "");
+				return d20plus.chat.emojiIndex && d20plus.chat.emojiIndex[clean] ? `[${clean}](https://github.com/TheGiddyLimit/emoji-dump/raw/master/out/${clean}.png)` : m1;
+			}));
+		}
+
+		let text = tc.val();
+		const srcText = text;
+
+		if (d20plus.cfg.getOrDefault("chat", "commands")) {
+			// add custom commands
+			text = text.replace(/^\/wb (.*?)$/gm, d20plus.chat.sendReply);
+			text = text.replace(/^\/ws (.*?)$/gm, d20plus.chat.sendToSelected);
+			text = text.replace(/^\/ttms( |$)/s, "/talktomyself$1");
+			text = text.replace(/^\/help(.*?)$/s, d20plus.chat.help);
+			if (!d20.textchat.talktomyself) text = text.replace(/^\/mtms ?(.*?)$/s, d20plus.chat.sendMyself);
+			if (is_gm) text = text.replace(/^\/v (.*?)$/s, d20plus.chat.requestScriptVersions);
+			if (d20plus.cfg.getOrDefault("chat", "languages")) text = text.replace(/\/in (.*?)$/gm, d20plus.chat.sendParsedInLanguage);
+			// text = text.replace(/^\/cl (on|off)$/sm, comprehendLanguages);
+		}
+
+		if (d20plus.cfg.getOrDefault("chat", "social")) {
+			const speakingto = $("#speakingto").val();
+			const speakingin = $("#speakingin").val();
+
+			if (speakingin) {
+				text = text.replace(/^[^/][^{^}]*?$/gm, msg => {
+					return d20plus.chat.sendInLanguage(msg, speakingin);
+				});
+			}
+
+			if (speakingto && speakingto !== "ttms") {
+				text = text.replace(/^([^/]*?)$/mgu, (...str) => {
+					const prepared = str[1].replace(/\/(r|roll) (?<dice>[ \dd+-]*)$/umg, "[[$<dice>]]");
+					return `/w "${speakingto}" ${prepared}`;
+				});
+			}
+		}
+
+		let toSend = $.trim(text);
```

Comment:
1. It was there before me
2. It mimics behavior of the r20 doChatInput, which does $.trim and not .trim()
I know $.trim is deprecated now, but since it's used in r20 and is slightly different from .trim() I'd prefer to keep it as is